code/game/objects/items/stunbaton.dm

```diff
@@ -34,6 +36,8 @@
 	var/preload_cell_type
 	///used for passive discharge
 	var/cell_last_used = 0
+	/// TESTING
+	var/obj/item/firing_pin/implant/mindshield/pin = /obj/item/firing_pin/implant/mindshield //standard firing pin for most guns
```

Comment:
This should probably default to an electronic pin, at the very least make the typepath of the pin var allow for it.